src/app/components/tile/index.html

```diff
@@ -27,6 +34,11 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Tile events">
+    <sky-demo-page-property
+      propertyName="helpClick"
```

Comment:
We probably want to call out that you have to have both `showHelp` set to true and be listening to `helpClick` in order for the help icon to appear. 

Settings works the same way.